src/index.css

```diff
@@ -84,6 +84,12 @@
 }
 
 @tailwind utilities;
+@layer components {
+  /* It's used to wrap main routes  */
+  .DateRangePicker__Month {
```

Comment:
1. Comment was about .wrapper, not about date range picker

2. Please don't place specific classes here, in global css. This class is used only in DateRangeSelector, so create DateRangeSelector.css and add `import './DateRangeSelector.css'` to the component file.

3. @layer components means that tailwind will generate a lot of classes based on your class, which is not needed in this case

4. there is a style for the same class below, outside of @layer components

Reply:
fixed